pom.xml

```diff
@@ -343,6 +343,11 @@
             <version>2.1</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-artifact</artifactId>
+            <version>3.0.3</version>
```

Comment:
@gumbelmj This dependency version is 5 years old, Rultor is using Maven 3.3.9 at this point, Maven Artifact is available in the same version [here](http://mvnrepository.com/artifact/org.apache.maven/maven-artifact/3.3.9).
If we're adding new dependencies we need to use the most recent versions unless there is a clear reason not to.